runtime/drivers/duckdb/transporter/sqlstore_to_duckDB.go

```diff
@@ -54,8 +54,7 @@ func (s *sqlStoreToDuckDB) Transfer(ctx context.Context, srcProps, sinkProps map
 		defer rowIter.Close()
 		return s.transferFromRowIterator(ctx, rowIter, sinkCfg.Table, opts.Progress)
 	}
-
-	limitInBytes := opts.LimitInBytes
+	limitInBytes, _ := s.to.(drivers.Handle).Config()["storage_limit_bytes"].(int64)
```

Comment:
Consider making this safe:
1. Turn `duckdb.connection.config` into `duckdb.Connection.Config`
2. `sqlStoreToDuckDB.to` can be made a `*duckdb.Connection` instead (since it's always a duckdb handle)
3. Then could do `s.to.Config.StorageLimitBytes`

Even if probably not required here, I think this is anyway a good change because we'll probably have more cases coming where the DuckDB transporters need access to DuckDB config values.

Reply:
Sure. We will need `duckdb.Connection.Configs` though since there is always a function `Config`

Reply:
Okay this is not possible since duckDB pkg imports transporter pkg.